conf/get_config.go

```diff
@@ -43,6 +50,18 @@ func GetProjectPath() (path string) {
 	panic("Unexpected error resolving this project's path in the global config file")
 }
 
+// GetGlobalProjectSettings returns the current global conf object for the current project
+func GetGlobalProjectSettings() (*Project, error) {
+	c := GetConfig()
+	for _, v := range c.Global.Projects {
+		if v.Name == c.Tokaido.Project.Name {
+			return &v, nil
+		}
+	}
+
+	return &Project{}, fmt.Errorf("unable to find global project configuration")
```

Comment:
Nitpick, maybe ok in context anway. unable -> Unable?

Reply:
"best practice" ... 🙄 

https://github.com/golang/go/wiki/CodeReviewComments#error-strings